src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/Filter/search_specification.rst

```diff
@@ -99,7 +102,7 @@ and filter with a ``bool`` query with ``must`` typed occurence of the following
             'bool' => [
                 'filter' => [
                     'match_phrase' => [
-                        'description-en_US-mobile-text' => '30 pages'
+                        'description-text-en_US-mobile' => '30 pages'
```

Comment:
What I remember is that we suffixes them with the backend types ? did it change since then ?